marklogic-data-hub-central/ui/src/components/sidebar/sidebar.tsx

```diff
@@ -380,15 +382,18 @@ const Sidebar: React.FC<Props> = (props) => {
     borderBottom: 'none',
     backgroundColor: '#F1F2F5'
   }
-
+     const callback = (key) => {
```

Comment:
@AkshayBajajML instead of having the name as callback function, could you name the function with what it exactly does